sdcm/nemesis.py

```diff
@@ -5246,6 +5246,37 @@ def disrupt_grow_shrink_zero_nodes(self):
         znode = random.choice([node for node in self.cluster.zero_nodes if node.dc_idx == self.target_node.dc_idx])
         self.decommission_nodes(nodes=[znode])
 
+    def disrupt_serial_restart_elected_topology_coordinator(self):
+        """ Rolling restart of topology coordinator node,
+        should trigger new coordinator node election
+        """
+        if not self.target_node.raft.is_consistent_topology_changes_enabled:
+            raise UnsupportedNemesis("Consistent topology changes feature is disabled")
+
+        self.use_nemesis_seed()
+        num_of_restarts = random.randint(1, len(self.cluster.nodes))
+        self.log.debug("Number of serial restart of topology coordinator: %s", num_of_restarts)
+        election_wait_timeout = random.choice([1, 5, 10, 15])
+        self.log.debug("Wait new topology coordinator election timeout: %s", election_wait_timeout)
+        self.unset_current_running_nemesis(self.target_node)
+        for _ in range(num_of_restarts):
+            coordinator_node = get_topology_coordinator_node(cluster=self.cluster)
```

Comment:
could reuse `new_coordinator_node` with:
`coordinator_node = new_coordinator_node or get_topology_coordinator_node(cluster=self.cluster) `

Reply:
i don't want to that way, because something could happened , and coordinator node will be reelected between iterations. Better to search again before restart